webrender/src/gpu_store.rs

```diff
@@ -5,7 +5,7 @@
 use renderer::MAX_VERTEX_TEXTURE_WIDTH;
 use std::mem;
 
-#[derive(Debug, Copy, Clone)]
+#[derive(Debug, Copy, Clone, Eq, Hash, PartialEq)]
```

Comment:
we may be able to store `ImageKey` instead of the gpu address in the cache key, but there is also `ClipAddressRange` in there that uses `GpuStoreAddress`